packages/plugins/rubygem-hammer_cli_foreman_kubevirt/rubygem-hammer_cli_foreman_kubevirt.spec

```diff
@@ -0,0 +1,98 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
+#
+# Generated from hammer_cli_foreman_kubevirt-0.0.1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
```

Comment:
Could you regenerate this with the correct template? In this case it's `hammer_plugin`.